package.json

```diff
@@ -11,7 +11,8 @@
     "build:history": "ROUTER_MODE=history npm run build",
     "test": "npm run update:lists && jest --clearCache && npm run test:unit",
     "codecov": "npm run test && codecov",
-    "dev": "npm run update && npm run update:mainlists && vue-cli-service serve --watch",
+    "juststart": "vue-cli-service serve --watch",
+    "dev": "npm run update && npm run juststart",
```

Comment:
and not sure why this was changed?

Reply:
[package.json](https://github.com/MyEtherWallet/MyEtherWallet/blob/v6/develop/package.json) in v6/develop